tests/vmi_multus_test.go

```diff
@@ -593,21 +579,41 @@ var _ = Describe("SRIOV", func() {
 
 	tests.BeforeAll(func() {
 		tests.BeforeTestCleanup()
-		// Create two sriov networks referring to the same resource name
+		// Extra cleaning of sriov networks
```

Comment:
Why only now? Please explain in the commit message, this has way more than alignment with the operator.

Reply:
It's part of the sriov-operator setup.

Now in the test we are creating SriovNetworks CRDs instead of NetworkAttachmentDefinition CRD's, and if we create an already existing one the creation will fail (so the second test will fail)

Reply:
Ok